src/drivers/px4fmu/fmu.cpp

```diff
@@ -328,10 +332,11 @@ PX4FMU::init()
 		return ret;
 
 	/* try to claim the generic PWM output device node as well - it's OK if we fail at this */
-	_class_instance = register_class_devname(PWM_OUTPUT_BASE_DEVICE_PATH);
+	ret = register_driver(PWM_OUTPUT0_DEVICE_PATH, &fops, 0666, (void *)this);
```

Comment:
Hm, ok, I don't really know what either of these two do, I rather looked at what's done in px4io.cpp.